recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "festim" %}
-{% set version = "2.0b0" %}
+{% set version = "2.0b1" %}
 {% set python_min = "3.10" %}
 {% set fenics_version = "0.10" %}
```

Comment:
hm in theory we also support dolfinx 0.9. Could we loosen the requirement below? to something like dolfinx>=0.9 <=0.10 ?